src/controllers/confirmation.controller.ts

```diff
@@ -124,6 +120,8 @@ const getOverdueFiling = ({isAccountsOverdue, isConfirmationStatementOverdue}):
     overdueFiling = "confirmation statement";
   } else if (isAccountsOverdue && isConfirmationStatementOverdue) {
     overdueFiling = "accounts and confirmation statement";
+  } else if (!isAccountsOverdue && !isConfirmationStatementOverdue) {
+    overdueFiling = "no open compliance case";
```

Comment:
overdueFiling no longer seems to be the right name to use for this variable and function name as 'no open compliance case' is not an overdue filing. Is it now just holding a general message? Couldn't see in the template changes below where it's used, so not sure...

Reply:
At the moment there is no handling for the scenario of when there's nothing overdue and the customer presses "no". This string isn't required at the moment but it's there just until a new story is drafted to cover that scenario. Otherwise it would just return the screen with an empty overdueFiling. If you think it should be removed entirely that's totally fair and I'll do that however, because I agree it doesn't really serve any purpose.

Reply:
Ok. I would just suggest maybe getting something like 'TBC' (To Be Completed) into the string that's output, then nice and clear that this is still a work in progress. I'd also add a comment to the code describing the above.

Reply:
a // TODO would be better, if we put 'TBC' into the string we'll forget about it!

Reply:
Agreed. I mean doing both. A // TODO comment and make that output text a bit clearer that it's really just a placeholder text for future functionality.